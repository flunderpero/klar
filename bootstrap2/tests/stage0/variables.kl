fn main():
    let string = "Ok"
    print(string)
    mut mutable = "Not Ok"
    mutable = "Ok: mutable"
    print(mutable)
    let i = 42
    let s = format("%i", i)
    print(s)
end
