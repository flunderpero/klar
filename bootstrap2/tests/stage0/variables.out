Ok
Ok: mutable
42
